examples/webgl_loader_mdd.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js WebGL - MDDLoader</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - MDDLoader
+		</div>
+
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+
+			import { MDDLoader } from './jsm/loaders/MDDLoader.js';
+
+			var camera, scene, renderer, mixer, clock;
+
+			init();
+
+			function init() {
+
+				scene = new THREE.Scene();
+
+				camera = new THREE.PerspectiveCamera( 35, window.innerWidth / window.innerHeight, 0.1, 100 );
+				camera.position.set( 10, 10, 10 );
+				camera.lookAt( scene.position );
+
+				clock = new THREE.Clock();
+
+				//
+
+				var hemiLight = new THREE.HemisphereLight( 0xffffff, 0x444444 );
+				hemiLight.position.set( 0, 20, 0 );
+				scene.add( hemiLight );
+
+				var dirLight = new THREE.DirectionalLight( 0xffffff );
+				dirLight.position.set( - 3, 10, - 10 );
```

Comment:
This light is shining from the back.

Depending on how you choose to fix this, try to avoid an over-bright scene.

Reply:
I thought it was intentional?

Reply:
It adds no light to the scene.

Reply:
Well, the top side is a little bit brighter with the directional light.

TBH, I think it's better to remove all lights and use `MeshNormalMaterial` instead.

Reply:
42c09065d951c526702b545fba016ce5e2a7f1e9